lib/Doctrine/Annotations/Metadata/Exception/InvalidAnnotationTarget.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Metadata\Exception;
+
+use InvalidArgumentException;
+use function sprintf;
+
+final class InvalidAnnotationTarget extends InvalidArgumentException
+{
+    public static function fromInvalidBitmask(int $bitmask) : self
```

Comment:
Tiny nitpick: this is the only factory method not named `new`. However, we can merge this as-is since I assume that we'll have to do a larger refactoring of exceptions to get them all to be consistent with these new exceptions.